tests/testthat/_snaps/backend-mysql.md

```diff
@@ -75,16 +75,3 @@
       SET `df_x`.`c` = COALESCE(`df_x`.`c`, `...y`.`c`), `df_x`.`d` = `...y`.`d`
       RETURNING `df_x`.`a`, `df_x`.`b` AS `b2`
 
-# can explain
```

Comment:
The snapshots are missing because you don't have a live database. Can you please add them back?
There is this nice git feature that lets you commit only part of a file which is very helpful for such cases.